cmd/notable-cli/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I have not intended to add a license. cobra init has done this. I will remove this file